scripts/snqi_sensitivity_analysis.py

```
@@ -0,0 +1,671 @@
+        score = (
+            weights.get("w_success", 1.0) * success
+            - weights.get("w_time", 1.0) * time_norm
+            - weights.get("w_collisions", 1.0) * coll
+            - weights.get("w_near", 1.0) * near
+            - weights.get("w_comfort", 1.0) * comfort
+            - weights.get("w_force_exceed", 1.0) * force_ex
+            - weights.get("w_jerk", 1.0) * jerk_n
+        )
+        return float(score)
+
+    def weight_sweep_analysis(
```

Pylint (3.3.7)

[refactor] Too many local variables (17/15)

(R0914)